cmd/evtest/main.go

```diff
@@ -34,6 +34,26 @@ func listDevices() {
 	}
 }
 
+func cloneDevice(dev *evdev.InputDevice) (*evdev.InputDevice, error) {
+	return evdev.CloneDevice(dev)
+}
+
+func createDevice() (*evdev.InputDevice, error) {
```

Comment:
Are these two some testing leftovers? I don't see any usages of `createDevice()` or `cloneDevice()` in evtest program.

Reply:
They were used for testing, i left them in thinking they might be useful as examples to people using the lib. I can remove them if you want though.

Reply:
zonque didn't raised any questions about it so it's fine I guess.

Reply:
I just missed that. You could just move those functions to a new example and actually call them. I would be in favor of no dead code in fact.

Reply:
@indeedhat, you've created new example file which is great, but it seems you forgot to delete mentioned functions from `cmd/evtest/main.go`.